woocommerce/woocommerce-gutenberg-products-block
# Temporarily disable the `woocommerce-blocks/product-hero-2-col-2-row` pattern

## What

Temporarily remove the `woocommerce-blocks/product-hero-2-col-2-row` pattern.

## Why


While testing #11208 we found the pattern was not working as expected, for more context see https://github.com/woocommerce/woocommerce-blocks/pull/11208#issuecomment-1759838220
We'll create a new issue to fix it and re-enable it.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new page or post.
2. Try to insert the `Product Hero 2 Column 2 Row` pattern and confirm it's not available.


## Screenshots or screencast



## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog

> Temporarily disable the "Product Hero 2 Column 2 Row" pattern.
